src/test/java/no/nav/tag/tiltaksgjennomforingprosess/factory/AvtaleTilPdfTest.java

```diff
@@ -10,6 +10,7 @@
 import org.apache.pdfbox.text.PDFTextStripper;
 import org.junit.Ignore;
 import org.junit.Test;
+import org.springframework.web.client.HttpServerErrorException;
```

Comment:
(Jeg får ikke til å kommentere utenfor endrede linje i fila)

`AvtaleTilPdf` har en privat metode som genererer `PDDocument`, og public metode som returnerer `byte[]`. I denne testen kalles altså metoden som returnerer `byte[]`. Deretter enkodes det til Base64-streng, dekodes til `byte[]` igjen, og konvertes tilbake til `PDDocument`. En veldig lang rundtur. Klassen burde returnere `PDDocument` på ikke-privat metode som kan kalles fra test.

Reply:
En base64 encodet byte array av pdf dokumentet var ment som en del av testen. Det er slik vi sender den til joark.

Reply:
Det som testes videre her er hva `PDDocument` inneholder, ikke `PDDocument -▶️ byte[]`. Men det er kanskje et poeng i seg selv å teste at prosessen er reversibel (`PDDocument -▶️ byte[] -▶️ PDDocument`?

Reply:
Ja, for å være helt sikker (men sikkert unødvendig). Uansett blir ikke dette kjørt i prod :)